plugins/processors/parser/parser.go

```diff
@@ -0,0 +1,129 @@
+package parser
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/parsers"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+type Parser struct {
+	Config      parsers.Config `toml:"config"`
```

Comment:
Use an embedded field for the parsers.Config:
```

type Parser struct {
	parsers.Config `toml:"config"`
```

This should make it so you can specify the options directly on the Parser object, similar to how it is set in input plugins:
```toml
[[processors.parser]]
  parse_fields = ["message"]
  original = "merge"
  data_format = "json"
  tag_keys = ["verb", "request"]
```